### PR TITLE
Feature/essifi 173

### PR DESCRIPTION
**Pull request by zoemaas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -555,4 +523,36 @@ HTTP status response: 500
...
+
+## For PEX-MS developers
+
+This project has been created using:
+* `yarn` version 1.22.11
+* `node` version 14.17.5
+* `docker-compose` version 1.26.2
+
+### Running
+1. Without docker:
```
Comment (sksadjad) [inline README.md:536]: without docker you need a mongodb instance installed

**Review by sksadjad — CHANGES_REQUESTED** [review]
all good just a note about the README

File: README.md (hunk 1 above)
Comment (zoemaas) [reply README.md:536]: Fixed!

**Review by sksadjad — APPROVED** [review]
looks good

**Pull request merged**